src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -277,6 +299,11 @@ _nns_edge_cmd_send (nns_edge_conn_s * conn, nns_edge_cmd_s * cmd)
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
 
+  if (!_nns_edge_check_connection (conn)) {
+    nns_edge_loge ("Failed to send command, socket has error.");
+    return NNS_EDGE_ERROR_IO;
+  }
```

Comment:
Since `_nns_edge_check_connection()` function checks whether `conn` object is NULL or not,so NULL checking in 292 line is redundant. What about removing this NULL checking code?

Reply:
IMO returning invalid-param if connection data is null, and io-error if socket is not available.

Reply:
Ah! I agreed with your opinion. :+1: